ext/lib/ipc/open-amp/open-amp/README.md

```diff
@@ -88,8 +88,8 @@ for Zephyr before building OpenAMP library for Zephyr.
 As Zephyr uses CMake, we build OpenAMP library as a target of Zephyr CMake
 project. Here is how to build libmetal for Zephyr:
 ```
-    $ export ZEPHRY_GCC_VARIANT=zephyr
-    $ export ZEPHRY_SDK_INSTALL_DIR=<where Zephyr SDK is installed>
+    $ export ZEPHYR_GCC_VARIANT=zephyr
```

Comment:
should be ZEPHYR_TOOLCHAIN_VARIANT

Reply:
Hah, thanks. I just was looking for your favorite typo and didn't notice it was using the old variable ;)